### ISSUE TITLE
About reconstruction loss and collision_rate

### ISSUE THREAD

**Issue by Wateau** [RUCAIBox/LC-Rec#20]
Thank you for sharing your code and detailed documentation. I am currently using your code and have encountered a few questions：
   You set the RQ-VAE training epoch to 5000, but in practice I noticed that the reconstruction loss and collision_rate tend to stabilize around 2k–3k epochs. Will continuing the training further improve performance or increase codebook usage? Also, do you have any suggestions for selecting the final model?
Thanks a lot!

**Comment by zhengbw0324** [comment]
@Wateau 
Hello, we did not conduct a careful hyperparameter search for the RQ-VAE training process in our experiment, so we may not be able to answer your question accurately. However, my intuition is that the number of epochs does not significantly affect the model performance.

**Comment by zhengbw0324** [comment]
For the final model, I typically use the model checkpoint of the 5000-th epoch in my experiments.

**Comment by Wateau** [comment]
During my training of the RQVAE model, I've observed that while the reconstruction loss decreases, the overall training loss gradually increases. Have you encountered this issue in your training process, and might it affect the quality of the generated item indices?

![Image](https://github.com/user-attachments/assets/37d4316c-cb33-417b-ba57-c779bee620b2)
